GNUmakefile

```diff
@@ -138,3 +139,7 @@ link-git-hooks: ## Install git hooks
 .PHONY: update-atlas-sdk
 update-atlas-sdk: ## Update the atlas-sdk dependency
 	./scripts/update-sdk.sh
+
+.PHONY: try-atlas-sdk-preview
+try-atlas-sdk-preview: ## Use SDK Preview, DONT MERGE
```

Comment:
If we really have to push this, this needs some sort of automated safety mechanism that would impede a push in master where we're using the sdk preview version. I can see how with a distraction we can incur in unexpected merges.

Reply:
in the meantime if your purpose is to test search index you can unblock yourself by testing this locally and syncing with the owning team on when is the plan of releasing these APIs in prod. I don't think we should tweak things and build workarounds to project timeline obstacles.

Reply:
@andreaangiolillo @marcosuma as @wtrocki is saying, this is to make it easier for people what wants/need to use the SDK Preview.   
In this way people don't need to do it manually. what problems do you see keeping this make target in case somebody wants to use it?  (there can always be a distraction and merge sdk preview changes even if this make target doesn't exist)

Reply:
> what problems do you see keeping this make target in case somebody wants to use it?

We are adding additional code to maintain for something that I think we should not use. If we need to use the SDK-preview, a better way which does not involve updating all the repo files is to use `replace =>` with the local SDK pointing to the SDK preview PR.

Reply:
only go.mod and go.sum are modified when using the SDK Preview

Reply:
Yes, it is true. One additional question: What would happen if the preview PR is not available on the SDK channel when running the script? Would the script fail, would the script run the github action on the atlas SDK?

I think you can move the preview logic to a new  PR so that we can keep discussing it there and unblock this PR